app/service/payments-service.ts

```diff
@@ -29,7 +29,7 @@ export default class PaymentService {
       case_type: 'Asylum',
       fees: [ fee ]
     };
-    req.app.locals.logger.trace(`Creating Card Payment with fee ${JSON.stringify(fee)}`, 'Payments Service');
+    req.app.locals.logger.trace(`Creating Card Payment with fee ${JSON.stringify(fee)} for appeal ref ${JSON.stringify(req.session.appeal.ccdCaseId)}`, 'Payments Service');
```

Comment:
@gavinslilley can you change the text 

`for appeal ref` to `for ccd id` instead.

(we don't actually have an IAC appeal reference at this stage).